pkg/cli/vault.go

```diff
@@ -88,19 +84,9 @@ func CheckVault(v *viper.Viper) error {
 		}
 
 		awsProfile := v.GetString(VaultAWSProfileFlag)
-		awsProfiles := []string{
-			VaultAWSProfileTranscomPPP,
-			VaultAWSProfileTranscomComLegacy,
-		}
-		if len(awsProfile) > 0 && !stringSliceContains(awsProfiles, awsProfile) {
-			return errors.Wrap(&errInvalidAWSProfile{Profile: awsProfile},
-				fmt.Sprintf("%s is invalid, expected %v", VaultAWSProfileFlag, awsProfiles))
-		}
-
-		// Require both are set or neither are set
-		if (len(keychainName) != 0 && len(awsProfile) == 0) || (len(keychainName) == 0 && len(awsProfile) != 0) {
-			return errors.Wrap(&errInvalidVault{KeychainName: keychainName, Profile: awsProfile},
-				fmt.Sprintf("If either %s or %s is set the other is required", VaultAWSKeychainNameFlag, VaultAWSProfileFlag))
+		if len(awsProfile) > 0 {
```

Comment:
Should this be 
```suggestion
		if len(awsProfile) == 0 {
```
Because you want to make sure that awsProfile DOES exists (IS > 0), right? Please correct if I'm misunderstanding.

Reply:
Off to the playground I go!

Reply:
https://play.golang.org/p/EALIjnmAO9l I think you're right. Did I do my example right?

Reply:
Yes -- I think you're understanding what I'm saying.